From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 26 Feb 2015 15:24:36 +0100
Subject: [PATCH] [media] coda: depend on HAVE_DMA_ATTRS for
 dma_alloc_writecombine

Allow to compile with ARCH_MXC disabled if HAVE_DMA_ATTRS is available.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
---
 drivers/media/platform/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/Kconfig b/drivers/media/platform/Kconfig
index ccbc9742cb7a..605493d54db7 100644
--- a/drivers/media/platform/Kconfig
+++ b/drivers/media/platform/Kconfig
@@ -137,8 +137,8 @@ if V4L_MEM2MEM_DRIVERS
 
 config VIDEO_CODA
 	tristate "Chips&Media Coda multi-standard codec IP"
-	depends on VIDEO_DEV && VIDEO_V4L2 && ARCH_MXC
-	depends on HAS_DMA
+	depends on VIDEO_DEV && VIDEO_V4L2 && (ARCH_MXC || COMPILE_TEST)
+	depends on HAS_DMA && HAVE_DMA_ATTRS
 	select SRAM
 	select VIDEOBUF2_DMA_CONTIG
 	select VIDEOBUF2_VMALLOC
